qiskit/visualization/tools/pi_check.py

```diff
@@ -45,7 +45,10 @@ def pi_check(inpt, eps=1e-6, output='text', ndigits=5):
         QiskitError: if output is not a valid option.
     """
     if isinstance(inpt, ParameterExpression):
-        return str(inpt)
+        try:
+            return pi_check(float(str(inpt)), eps=eps, output=output, ndigits=ndigits)
```

Comment:
```suggestion
            return pi_check(float(inpt), eps=eps, output=output, ndigits=ndigits)
```
If you're looking to check if the Parameter is fully bound, you can call `float()` on it directly (which will raise a `TypeError` if it is not fully bound).